src/form/AddCardCompleteForm.tsx

```diff
@@ -0,0 +1,64 @@
+import {useContext, useState} from "react";
+import CardContext from "../context/CardContext.tsx";
+import Display from "../components/Display.tsx";
+import Input from "../components/Input.tsx";
+import AddCardComplete from "../components/AddCardComplete.tsx";
+
+function AddCardCompleteForm() {
+    const {cards} = useContext(CardContext)
+    const [cardAlias, setCardAlias] = useState('')
+    const card = cards[cards.length-1]
```

Comment:
r: 배열이 비어있을 경우 에러가 발생하는 코드입니다! 보다 안전한 방법으로 가져올 수 있도록 예외 처리해주세요

Reply:
예외처리 추가했습니다.

Reply:
a: 옵셔널체이닝 혹은 trycatch로도 가능한 점 참고해주세요

```suggestion
    const card = cards?.[cards.length-1]
```